script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_5B.txt

```diff
@@ -5,26 +5,26 @@ Arcueid descended from a height of 80 meters above Vlov.
 No, it was more like she flew.
 While in the air, she took aim and angled herself before accelerating like a missile.
 <Page3>
-Sand and debris spewed up in the air.
-The wind whipped up a cloud of dust that sweeps the area.
-The river of flame that spread across the downtown area is eradicated by that single blow.
+Sand and debris spews up in the air.
```

Comment:
"spew" is the correct present tense here.

Reply:
Sand and debris billow up into the air.

(but I just hate usage of 'spew' for anything that isn't toxic/gross, the mental linkage is just too strong for most readers)

Reply:
I disagree about the mental imagery, but I don't really mind billow so that's fine with me